app/models/knight.rb

```diff
@@ -1,2 +1,19 @@
 class Knight < Piece
+  def obstructed_diagonally?
```

Comment:
Originally I included `(to_x:, to_y:)` at the end of each `def obstructed_` line, but Rubocop gave me a warning for each:
```app/models/knight.rb:2:30: W: Lint/UnusedMethodArgument: Unused method argument - to_x. You can also write as obstructed_diagonally?(*) if you want the method to accept any arguments but don't care about them. (https://github.com/bbatsov/ruby-style-guide#underscore-unused-vars)
  def obstructed_diagonally?(to_x:, to_y:)
                             ^^^^
app/models/knight.rb:2:37: W: Lint/UnusedMethodArgument: Unused method argument - to_y. You can also write as obstructed_diagonally?(*) if you want the method to accept any arguments but don't care about them. (https://github.com/bbatsov/ruby-style-guide#underscore-unused-vars)
  def obstructed_diagonally?(to_x:, to_y:)
                                    ^^^^
app/models/knight.rb:6:30: W: Lint/UnusedMethodArgument: Unused method argument - to_x. You can also write as obstructed_vertically?(*) if you want the method to accept any arguments but don't care about them. (https://github.com/bbatsov/ruby-style-guide#underscore-unused-vars)
  def obstructed_vertically?(to_x:, to_y:)
                             ^^^^
app/models/knight.rb:6:37: W: Lint/UnusedMethodArgument: Unused method argument - to_y. You can also write as obstructed_vertically?(*) if you want the method to accept any arguments but don't care about them. (https://github.com/bbatsov/ruby-style-guide#underscore-unused-vars)
  def obstructed_vertically?(to_x:, to_y:)
                                    ^^^^
app/models/knight.rb:10:32: W: Lint/UnusedMethodArgument: Unused method argument - to_x. You can also write as obstructed_horizontally?(*) if you want the method to accept any arguments but don't care about them. (https://github.com/bbatsov/ruby-style-guide#underscore-unused-vars)
  def obstructed_horizontally?(to_x:, to_y:)
                               ^^^^
app/models/knight.rb:10:39: W: Lint/UnusedMethodArgument: Unused method argument - to_y. You can also write as obstructed_horizontally?(*) if you want the method to accept any arguments but don't care about them. (https://github.com/bbatsov/ruby-style-guide#underscore-unused-vars)
  def obstructed_horizontally?(to_x:, to_y:)```